index.html

```diff
@@ -33,9 +33,9 @@ <h4 class="service-heading">Cloud Architecture</h4>
           <i class="fa fa-circle fa-stack-2x text-primary"></i>
           <i class="fa fa-code fa-stack-1x fa-inverse"></i>
         </span>
-        <h4 class="service-heading">Puppet Module Development</h4>
+        <h4 class="service-heading">Puppet Consulting</h4>
         <p class="text-muted">
-          We specialize in building Puppet modules to meet your specific needs.
+          We specialize in <a href="./puppet_consulting/">building Puppet modules to meet your specific needs.</a>
```

Comment:
I think it'd be good, if we do keep this Puppet-related blurb here, to expand it to mention Puppet infrastructure as well as Puppet modules.

For example, Puppet infrastructure scaling, tuning, monitoring, etc...

Reply:
Oh, woops... I didn't see that you do call that out in the Puppet-specific page below.

So perhaps then just simplifying the original text of:
> We specialize in building Puppet modules to meet your specific needs.
> Have a custom Java app that needs to be deployed more quickly and reliably?  No problem!  We're here to help you fully automate the  process.

Down into something like:
> We specialize in Puppet infrastructure and module development. Having Puppetserver scaling problems or need a module for your custom app? No problem! We're here to help.


...or something like that.

Reply:
Yeah, that link was a proof of concept to figure out how to link to the new page I created. It definitely needs cleaned up once we finalize what's on the consulting page. I'll fix this up once we're happy with the consulting page.